docs/index.rst

```diff
@@ -54,6 +54,7 @@ The vision is that it grows over time as the teams and governance structure evol
    :hidden:
 
    community_leadership/community_leadership
+   community_leadership/community_council
```

Comment:
I'm not sure if this should be included in the `index.rst` as "Community Council" is a subsection of "Community Leadership"